Tasks/PackerBuildV1/task.json

```diff
@@ -245,6 +245,15 @@
             "defaultValue": "",
             "helpMarkDown": "Provide a name for the output variable which will store generated machine image VHD url for un-managed VM image or image name for managed VM image.",
             "groupName": "Output"
+        },
+        {
+            "name": "imageId",
+            "type": "string",
+            "label": "Azure Resource Id",
+            "required": false,
+            "defaultValue": "",
+            "helpMarkDown": "Provide a name for the output variable which will store the azure resource id for the newly created image.",
```

Comment:
@pulkitaggarwl @azooinmyluggage @bishal-pdMSFT also this !

Reply:
Here is a screenshot
![image](https://user-images.githubusercontent.com/37842906/50690999-6ceb1900-0ffd-11e9-9152-b28dbd0c17bc.png)